[2022-02-19 15:06:01,659] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.clean_folder manual__2022-02-19T15:05:46.942408+00:00 [queued]>
[2022-02-19 15:06:01,669] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.clean_folder manual__2022-02-19T15:05:46.942408+00:00 [queued]>
[2022-02-19 15:06:01,669] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 15:06:01,670] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 15:06:01,670] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 15:06:01,683] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): clean_folder> on 2022-02-19 15:05:46.942408+00:00
[2022-02-19 15:06:01,688] {standard_task_runner.py:52} INFO - Started process 13638 to run task
[2022-02-19 15:06:01,697] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'clean_folder', 'manual__2022-02-19T15:05:46.942408+00:00', '--job-id', '402', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmpc5v2fav5', '--error-file', '/tmp/tmp46ladpby']
[2022-02-19 15:06:01,698] {standard_task_runner.py:77} INFO - Job 402: Subtask clean_folder
[2022-02-19 15:06:01,769] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.clean_folder manual__2022-02-19T15:05:46.942408+00:00 [running]> on host d18a6d5a39bc
[2022-02-19 15:06:01,825] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=clean_folder
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T15:05:46.942408+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T15:05:46.942408+00:00
[2022-02-19 15:06:01,826] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-19 15:06:01,827] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'find /opt/***/data -name "*.csv" -type f -delete']
[2022-02-19 15:06:01,839] {subprocess.py:85} INFO - Output:
[2022-02-19 15:06:01,844] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-19 15:06:01,871] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=clean_folder, execution_date=20220219T150546, start_date=20220219T150601, end_date=20220219T150601
[2022-02-19 15:06:01,909] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 15:06:01,944] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
